### PR TITLE
Add support for ACL on objects

### PR DESCRIPTION
**Pull request by flecno**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,12 @@ The region the AWS `bucket` is located in.
 
 *Default:* `us-east-1`
 
+### acl
+
+The ACL to apply to the objects.
+
+*Default:* `public-read`
```
Comment (achambers) [inline README.md:90]: @flecno I can't see where you are setting the default value in the code. Can you please follow the conventions of the other default values in this file?

Comment (flecno) [inline README.md:90]: You are right. It's not really explicit. The default already exists in https://github.com/ember-cli-deploy/ember-cli-deploy-s3/blob/59e083a73141bf17e67a83981ded573ff8da9caf/lib/s3.js#L68

But I would prefer to move the default to another place too. I will look into the other defaults.

Comment (achambers) [inline README.md:90]: Cool. If you're going to make it a part of the plugin config then the
default should be moved out of there and into the index.js. Let me know
when you've updated it and I'll take a look.

Thanks for your contributions.

**Comment by flecno** [comment]
@achambers here is an update. What do you say? :)

**Comment by flecno** [comment]
I rebased this to latest master. Can you review this please?

**Comment by ghedamat** [comment]
@flecno LGTM, thanks!

**Pull request merged**